utils/timezone.jsx

```diff
@@ -6,8 +6,11 @@ import moment from 'moment-timezone';
 
 import store from 'stores/redux_store.jsx';
 
-export function getSupportedTimezones() {
-    return getTimezones(store.getState());
+// In user_settings the state is being passed here.
+// In timezone_provider it is not.
+// Not sure how to refactor the timezone_provider yet.
```

Comment:
We can create a followup ticket for this if you'd like

Reply:
Yes I will make a note of it, as well as the 'localizeMessage' function.

Reply:
Ticket: https://mattermost.atlassian.net/browse/MM-21160